index.html

```diff
@@ -93,14 +93,14 @@ <h2 class="page-header"><i class="fa fa-cubes"></i> Projects</h2>
      <ul>
      <li><a href="https://github.com/JuliaReach/LazySets.jl/">LazySets.jl</a> A Julia package for calculus with convex sets.</li>
      <li><a href="https://github.com/JuliaReach/Reachability.jl/">Reachability.jl</a> Reachability and Safety of Nondeterministic Dynamical Systems.</li>
-     <li><a href="https://github.com/JuliaReach/MathematicalSystems.jl/">MathematicalSystems.jl</a> Systems definitions in Julia.</li>
```

Comment:
There is a sentence in the paragraph above about `MathematicalSystems` which should be (re)moved I think.

Reply:
True; it is more of a (julian) cultural thing to use "... in Julia" , ".. for Julia..", "A Julia ..." in the package's description, and i like it, but we are not bound to that convention.

Reply:
(*) this is seen by exploring a bit https://pkg.julialang.org/docs/